src/content/blog/posts/iceberg-athena-dbt.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Building a transaction data lake using Amazon Athena, Apache Iceberg and dbt
+description: How we leveraged Amazon Athena, along with the Apache Iceberg table format and the dbt SQL management framework, to build robust, scalable and maintainable ELT (extract, load, transform) pipelines.
+date: 2024-18-07
+author:
+  name: Dr Soumaya Mauthoor
+permalink: "/blog/posts/{{ title | slugify }}/"
+tags:
+  - Athena
+  - Iceberg
+  - dbt
+---
+
+At the [UK Ministry of Justice (MoJ)](https://mojdigital.blog.gov.uk/), our goal is to make the justice system simpler and quicker while saving public money. The [MoJ Analytical Platform](https://user-guidance.analytical-platform.service.justice.gov.uk/) supports this mission by facilitating data sharing and analysis within the MoJ, aiding better decision-making.
```

Comment:
I'm not sure we can claim that as the goal of MoJ... you could say "_a goal of the digital and data professions in MoJ_" instead?